hatchery/transitgateway.go

```diff
@@ -503,16 +503,20 @@ func (creds *CREDS) acceptTGWShare() error {
 	}
 
 	if len(resourceShareInvitation.ResourceShareInvitations) == 0 {
+		Config.Logger.Print("No pending ResourceShareInvitations")
 		return nil
 	} else {
-		if *resourceShareInvitation.ResourceShareInvitations[0].Status != "ACCEPTED" {
-			_, err := svc.AcceptResourceShareInvitation(&ram.AcceptResourceShareInvitationInput{
-				ResourceShareInvitationArn: resourceShareInvitation.ResourceShareInvitations[0].ResourceShareInvitationArn,
-			})
-			if err != nil {
-				return err
+		for _, rsi := range resourceShareInvitation.ResourceShareInvitations {
+			Config.Logger.Printf("ResourceShareInvitation: %s", *rsi)
+			if *rsi.Status != "ACCEPTED" {
+				_, err := svc.AcceptResourceShareInvitation(&ram.AcceptResourceShareInvitationInput{
+					ResourceShareInvitationArn: rsi.ResourceShareInvitationArn,
+				})
+				if err != nil {
+					return err
+				}
+				return nil
```

Comment:
should it return here? or should this for loop keeps moving until all RSIs have been checked?

Reply:
I think we just want to accept any pending RSI's. Let me revisit my logic here a little bit, I don't remember my thought process entirely at this point

Reply:
i feel like this is a different fix/feature. Could we move it to a different PR so we can merge the string length fix that is blocking CI?